.github/workflows/daily.yaml

```diff
@@ -21,8 +21,9 @@ jobs:
       - name: Reclaim Disk Space
         run: .github/ci-prerequisites.sh
       - name: Install JDK {{ matrix.java }}
-        uses: joschi/setup-jdk@e87a7cec853d2dd7066adf837fe12bf0f3d45e52
+        uses: actions/setup-java@v3
         with:
+          distribution: 'adopt-hotspot'
```

Comment:
`Adopt OpenJDK` is moved to Eclipse Temurin, [see](https://github.com/actions/setup-java/blob/v2/README.md#supported-distributions).

So it won't be updated anymore, I think we should used `temurin` as it is also the one we have defined in our test-suite. For instance, [here](https://github.com/quarkus-qe/quarkus-test-suite/blob/main/.github/workflows/daily.yaml#L20).

Reply:
OK, wanted to have diversity across our TS, but will move to `temurin`, same as the main TS